spec.html

```diff
@@ -3720,6 +3731,8 @@
           </tbody>
         </table>
       </emu-table>
+      <!-- If ToObject is ever made hookable, we need to alert the editors of the HTML Standard as
+           that'll effect the security model of the Location exotic object. -->
```

Comment:
Yes, the `@@toPrimitive` example I mentioned. But also `@@toStringTag`, `@@hasInstance`, and `@@isConcatSpreadable` require special behavior for cross-origin objects. Basically any change where before we just followed an internal code path and now we need to call into userland.